File: Pwa/src/app/models/viewModel/facilityWiseIndicatorViewModel.ts
import { indicatorGetViewModel } from './indicatorGetViewModel';


export class facilityWiseIndicatorViewModel{
    facilityId:number;
    facilityName:string;
    instanceId:number;
    indicators:indicatorGetViewModel[]
}

export class BeneficiaryWiseIndicatorViewModel{
    facilityId:number;
    facilityName:string;
    instanceId:number;
    indicators:indicatorGetViewModel[]
}

File: Api/src/UnicefEducationMIS.Core/ViewModel/FacilityWiseIndicatorViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UnicefEducationMIS.Core.ViewModel
{
    public class FacilityWiseIndicatorViewModel
    {
        public long FacilityId { get; set; }
        public string FacilityName { get; set; }
        public long InstanceId { get; set; }
        public List<IndicatorGetViewModel> Indicators { get; set; }
    }
}
